### PR TITLE
bsnes: rename bsnes to bsnes2014 and bsnes_hd to bsnes

### PR DESCRIPTION
**Pull request by natinusala**

### CODE REVIEW

**Comment by ToKe79** [comment]
Hi,
the build of this PR (f10bd21221a47b70021a2f91ac3ea496ae03a12b) finished successfully for all 46 targets.
Images for testing and logs are here: http://nightly.builds.lakka.tv/PRs/pr_821_000109/

_This is an automated message_

File: projects/RPi/options [hunk 1]
```diff
@@ -148,7 +148,7 @@
     LIBRETRO_CORES="${LIBRETRO_CORES// beetle-psx /}"
     LIBRETRO_CORES="${LIBRETRO_CORES// beetle-saturn /}"
     LIBRETRO_CORES="${LIBRETRO_CORES// beetle-vb /}"
-    LIBRETRO_CORES="${LIBRETRO_CORES// bsnes_hd /}"
+    LIBRETRO_CORES="${LIBRETRO_CORES// bsnes2014 /}"
```
Comment (ToKe79) [inline projects/RPi/options]: shouldn't it be ` bsnes ` which should be removed from the list?

Comment (natinusala) [reply projects/RPi/options]: Indeed, let me fix it

Comment (natinusala) [reply projects/RPi/options]: Actually both are removed, see line 153

**Pull request merged**